extensions/ql-vscode/src/interface.ts

```diff
@@ -390,7 +393,7 @@ export class InterfaceManager extends DisposableObject {
       resultsPath: this.convertPathToWebviewUri(
         results.query.resultsPaths.resultsPath
       ),
-      parsedResultSets: await getParsedResultSets(),
+      parsedResultSets,
```

Comment:
I was going to ask why you inlined this function, since obviously it was called in multiple places... but I see that it's not. Maybe it was in the past. Agree that inlining it now makes sense.